backend/telemetry/src/main/java/org/sonarsource/sonarlint/core/telemetry/TelemetryLocalStorage.java

```diff
@@ -276,6 +293,15 @@ public void incrementShowIssueRequestCount() {
     showIssueRequestsCount++;
   }
 
+  public void incrementFixSuggestionReceivedCount(String suggestionId) {
+    markSonarLintAsUsedToday();
+    this.fixSuggestionReceivedCounter.computeIfAbsent(suggestionId, k -> new TelemetryFixSuggestionReceivedCounter()).incrementFixSuggestionReceivedCount();
+  }
+
+  public void fixSuggestionResolved(String suggestionId, FixSuggestionStatus status, @Nullable Integer snippetIndex) {
```

Comment:
In theory, it is possible that user opens the suggestion one day, and resolves it the other day :D 
So I think we should call `markSonarLintAsUsedToday();` here as well.

Reply:
Right!